run_dir/design/ont_flowcells.html

```diff
@@ -171,6 +180,14 @@ <h1>
                         {% end %}
                     </td>
 
+                    <td class="first_mux_loss text-center">
+                        {% try %}
+                            {{ ont_flowcells[onefc]["first_mux_loss_pc"] }} 
+                        {% except %}
+                            {{ "" }}
+                        {% end %}
```

Comment:
Could we not have `ont_flowcells[onefc].get("first_mux_loss_pc", "")` here instead of a try except?